airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/RefreshSchemaActivityImpl.java

```diff
@@ -17,9 +20,11 @@
 public class RefreshSchemaActivityImpl implements RefreshSchemaActivity {
 
   private final Optional<ConfigRepository> configRepository;
+  private final SchedulerHandler schedulerHandler;
 
-  public RefreshSchemaActivityImpl(Optional<ConfigRepository> configRepository) {
+  public RefreshSchemaActivityImpl(Optional<ConfigRepository> configRepository, SchedulerHandler schedulerHandler) {
```

Comment:
We should not use this. Why not using the SourceApi to actually call the API, this won't call the API